src/plugins/ui_core_views/pivot_ui.ts

```diff
@@ -0,0 +1,271 @@
+import { Token } from "../../formulas";
+import { astToFormula } from "../../formulas/parser";
+import {
+  UNDO_REDO_PIVOT_COMMANDS,
+  getFirstPivotFunction,
+  getNumberOfPivotFunctions,
+} from "../../helpers/pivot/pivot_helpers";
+import { pivotRegistry } from "../../helpers/pivot/pivot_registry";
+import { Pivot } from "../../helpers/pivot/pivot_runtime";
+import {
+  AddPivotCommand,
+  CellPosition,
+  Command,
+  CoreCommand,
+  UID,
+  UpdatePivotCommand,
+} from "../../types";
+import { UIPlugin, UIPluginConfig } from "../ui_plugin";
+
+function isPivotCommand(cmd: CoreCommand): cmd is AddPivotCommand | UpdatePivotCommand {
+  return UNDO_REDO_PIVOT_COMMANDS.includes(cmd.type);
```

Comment:
maybe co-locate `UNDO_REDO_PIVOT_COMMANDS` next to `isPivotCommand` here if it's used only once here.

And export `isPivotCommand` in `index.ts` instead ?